docs/sources/developer/Developers-Guide.md

```diff
@@ -27,6 +27,7 @@ Once you've set up your development environment we recommend start simple:
 * [How to write a parser plugin](How-to-write-a-parser-plugin.md)
 * [How to write an analysis plugin](How-to-write-an-analysis-plugin.md)
 * [How to write an output module](How-to-write-an-output-module.md)
+* [How to write a tagging rule](How-to-write-a-tagging-rule.html)
```

Comment:
The file you're adding is .md, why the changes to .html?
Is there some github magic happening, or is this because these are supposed to be hosted elsewhere?

Reply:
I think the magic is here https://github.com/log2timeline/plaso/tree/master/docs, I need to dig into this a bit for some other PRs

Reply:
it looks to me that we still want to link to .md here, and the link get changed to .html later on by some magic

Reply:
ack, I'll change it to .md, in parallel I've started working on sphinx-doc CI tests to catch these issues.